src/components/JaegerIntegration/JaegerResults/JaegerTraceTitle.tsx

```diff
@@ -30,6 +30,7 @@ export class JaegerTraceTitle extends React.Component<JaegerScatterProps> {
                 href={this.props.onClickLink}
                 style={{ right: '130px', fontSize: '16px', position: 'absolute' }}
                 target={'_blank'}
+                rel="noopener noreferrer"
```

Comment:
???

Reply:
This was  security warning that was actually failing CI.  Apparently you need this when using target={'_blank'}.